stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -2655,6 +2655,10 @@ function(_add_swift_target_executable_single name)
     set_target_properties(${name} PROPERTIES
       BUILD_WITH_INSTALL_RPATH YES
       INSTALL_RPATH "@executable_path/../lib/swift/${SWIFT_SDK_${SWIFTEXE_SINGLE_SDK}_LIB_SUBDIR};@executable_path/../../../lib/swift/${SWIFT_SDK_${SWIFTEXE_SINGLE_SDK}_LIB_SUBDIR}")
+  elseif(SWIFT_HOST_VARIANT_SDK STREQUAL "LINUX")
+    set_target_properties(${name} PROPERTIES
+      BUILD_WITH_INSTALL_RPATH YES
+      INSTALL_RPATH "$ORIGIN/../../../lib/swift/${SWIFT_SDK_${SWIFTEXE_SINGLE_SDK}_LIB_SUBDIR}")
```

Comment:
This will add this relative rpath to _all_ target executables on linux, like `swift-reflection-test`, so it seems mistaken. Better to [pass in a relative path, like `_add_swift_runtime_link_flags()` does with `relpath_to_lib_dir`](https://github.com/apple/swift/blob/7b460ce4951c32dbcc0bdeecf1bbb1f67075d429/cmake/modules/AddSwift.cmake#L444).

Reply:
True, it isn't installed now.